scriptmodules/supplementary/emulationstation.sh

```diff
@@ -124,4 +130,8 @@ _EOF_
     mkdir -p "/etc/emulationstation"
 
     configure_inputconfig_emulationstation
-}
+
+    # run sudo without password so emulationstation can shutdown and restart system
+    local file="/etc/sudoers"
+    grep -q "$user" $file || echo "$user ALL=(ALL) NOPASSWD:ALL" >> $file
```

Comment:
We already discussed this - accident or ?

You should use sudoers.d btw (but not as part of the ES module)